src/helpers/query/assignments.js

```diff
@@ -203,7 +203,7 @@ export const assignmentCounter = (adminId, orgType, orgId) => {
   })
 }
 
-export const scoresPageFetcher = async (adminId, orgType, orgId, pageLimit, page, skinny = false, paginate = true) => {
+export const assignmentPageFetcher = async (adminId, orgType, orgId, pageLimit, page, includeScores = false, skinny = false, paginate = true) => {
```

Comment:
It appears that `includeScores` is not used. Shouldn't the `scoresPromises` be wrapped in `if (includeScores)` or something like that?